.drone/drone.yml

```diff
@@ -36,6 +36,8 @@ trigger:
   ref:
   - refs/heads/main
   - refs/tags/**
+  - refs/heads/r? # For weekly release
+  - refs/heads/r?? # For weekly release
```

Comment:
How does the `r?` and `r??` work and how is this different from `**`? I can't find this in their docs.

Reply:
From https://docs.drone.io/pipeline/triggers/

> You can use wildcard matching in your triggers. Note that triggers use glob pattern matching, not regular expressions.

So it will match on `r1`, `r33`, etc. We could change this to `refs/heads/r*`, which will build every branch that starts with `r`.